### DISCUSSION TITLE
Problem downloading the aspire nuget packages

### DISCUSSION THREAD

**Discussion by mip1983** [microsoft/aspire#2620] [General]
I'm not sure if this is an issue with the aspire nuget packages, or nuget and preview packages in general, but friday last week and today I'm having issue's with the aspire nuget packages. It went on for a while on friday then everything just came back n started working, then it's broken again today.

At first I though it was a local machine issue, something with VS22 or a project issue, but I've tried all the usual nuget cache clearing, and  on a different machine on the file > new aspire start template and I'm getting it there too.

At first it looks like all packages are problematic with an exclamation mark on. But if I remove them all and start adding them back one by one, it's when I add the aspire preview packages that I start to hit the issue.
It's a weird one, the packages look to be downloaded and the app still builds, but yellow exclamation marks are there on the preview packages, and so I get no intellisense and/or lots of red underlining on code that references these packages.

Once it starts happening with the preview ones, it seems to happen to any subsequent package you install:

![image](https://github.com/dotnet/aspire/assets/2116108/b4789a3e-4215-415f-80e5-03d4c30de7cd)

I see this in the error list when I add/remove packages:

![image](https://github.com/dotnet/aspire/assets/2116108/25e8c552-3544-4705-b21a-28508fcccaa3)

Which points to this chunk of XML in Microsoft.WebTools.Aspire.targets:

```
  <Target Name="GenerateCombinedAppSettingsSchemaTarget"
          DependsOnTargets="GenerateCombinedComponentSchemaTarget"
          Condition=" '@(JsonSchemaSegment)' != '' "
          BeforeTargets="CompileDesignTime"
          Inputs="$(SchemaGenIncrementalBuildInputs)"
          Outputs="$(AppSettingsJsonSchemaCombinedFilePath)">
    <GenerateCombinedAppSettingsSchema
      AppSettingJsonSchemaFilePath="$(AppSettingJsonSchemaLoadFilePath)"
      ComponentJsonSchemaCombinedFilePath="$(JsonSchemaCombinedFilePath)"
      AppSettingsJsonSchemaCombinedFilePath="$(AppSettingsJsonSchemaCombinedFilePath)">
    </GenerateCombinedAppSettingsSchema>
    <Message Text="Generating appsettingsschema" />
  </Target>
```

Does anyone know if there's something going on with nuget and/or the aspire preview packages?

**Comment by dkurepa** [comment]
I have the same issue in VS. Writing in VS Code and building with the command line is still working for me tho

**Reply by premun** [reply]
Repro steps easy:
1. `git clone https://github.com/dotnet/arcade-services`
2. `cd arcade-services && start arcade-services.sln`
3. F5
